experimental/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ All notable changes to experimental packages in this project will be documented
 
 ### :boom: Breaking Change
 
+* chore(metrics-sdk): clean up exports [#3197](https://github.com/open-telemetry/opentelemetry-js/pull/3197) @pichlermarc
+  * removes export for:
+    * `AccumulationRecord`
+    * `Aggregator`
+    * `AggregatorKind`
+    * `Accumulation`
+    * `CollectionResult`
```

Comment:
I believe `CollectionResult` is a public interface too, as it is the return value of `MetricReader.collect`.

Reply:
It is. Was too eager in removing it. Added back in [291470c](https://github.com/open-telemetry/opentelemetry-js/pull/3197/commits/291470cbccbd7a3efe0566723e23204fc23ffe72). Thanks! :slightly_smiling_face: